### ISSUE TITLE
import using SampleModule.forRoot()

### ISSUE THREAD

**Issue by MadUser** [jvandemo/generator-angular2-library#92]
In your example you import the sample module to the app module using:
```
imports: [
    BrowserModule,
    
    // Specify your library as an import
    SampleModule
  ],
```
This cause me "no provider error".
When I use forRoot, it solve the problem:
```
imports: [
    BrowserModule,
    
    // Specify your library as an import
    SampleModule.forRoot()
  ],
```
Btw,
Good work. really helpful project.

**Comment by jvandemo** [comment]
@MadUser &mdash; Thank you for reporting. I have updated the README file 👍 

PS: thanks for the kind words!

**Comment by Kangcor** [comment]
Please, remember to update the NPMJS readme in order to keep the docs in sync ([here](https://www.npmjs.com/package/generator-angular2-library#consuming-your-library) the .forRoot() isn't specified.

Thank you for this awesome project, you just save me a lot of hours trying to make a library.

**Comment by jvandemo** [comment]
@Kangcor &mdash; I have released v10.1.1 to update the documentation on the NPM registry.

Thank you for reporting! 👍
